### DISCUSSION TITLE
case stopped due to large energy error

### DISCUSSION THREAD

**Discussion by Taylor1987** [OpenRadioss/OpenRadioss#3260] [Q&A]
Hi, 
I have a question on the frequently suffered before and now. 
the program is stopped due to the energy err, from the mesaage i guess maybe the cause below, DM/Mass0 is too large, and even have found the node, where it's belong to the tie surface (the elem quality is not too bad), I have set the type2 as the best practice guideline

      NODE       MASS      MASS0   DM/MASS0
     15847 0.5024E-08 0.7095E-11  707.2

from the part of 0001.out below how can i resolve such issue? 
sorry , maybe it's tricky to answer, as this is a real ongoing project, I can't upload the full rad file to this forum.
apprreciated if someone can give me clue to solve it.
Thanks advanced~

part of information below:

 SENSOR NUMBER          2 ACTIVATED AT TIME  0.00000E+00
 22/12/2024
   CYCLE    TIME      TIME-STEP  ELEMENT          ERROR  I-ENERGY    K-ENERGY T  K-ENERGY R  EXT-WORK     MAS.ERR     TOTAL MASS  MASS ADDED
       0   0.000      0.1000E-06 NODE        7308   0.0%   0.000       0.000       0.000       0.000      0.1356      0.2446E-03  0.2920E-04

    5000  0.5000E-03  0.1000E-06 NODE     1844439   0.0%   7.861       21.75       0.000       29.61      0.1356      0.2446E-03  0.2920E-04
   10000  0.1000E-02  0.1000E-06 NODE        7406   0.0%   7.041       25.13       0.000       32.17      0.1356      0.2446E-03  0.2920E-04
   15000  0.1500E-02  0.1000E-06 NODE        7406   0.0%   5.515       14.43       0.000       19.94      0.1356      0.2446E-03  0.2920E-04
   20000  0.2000E-02  0.1000E-06 NODE        7406   0.0%   7.185       21.24       0.000       28.43      0.1356      0.2446E-03  0.2920E-04
   25000  0.2500E-02  0.1000E-06 NODE        7406   0.0%   6.224       24.41       0.000       30.63      0.1356      0.2446E-03  0.2920E-04
   30000  0.3000E-02  0.1000E-06 NODE        7406   0.0%   6.991       15.14       0.000       22.13      0.1356      0.2446E-03  0.2920E-04
   35000  0.3500E-02  0.1000E-06 NODE        7406   0.0%   7.134       17.55       0.000       24.69      0.1356      0.2446E-03  0.2920E-04
   40000  0.4000E-02  0.1000E-06 NODE        7406   0.0%   6.012       22.61       0.000       28.62      0.1356      0.2446E-03  0.2920E-04
   45000  0.4500E-02  0.1000E-06 NODE        7406   0.0%   7.299       17.30       0.000       24.60      0.1356      0.2446E-03  0.2920E-04
   50000  0.5000E-02  0.1000E-06 NODE        7406   0.0%   6.369       18.21       0.000       24.58      0.1356      0.2446E-03  0.2920E-04

   55000  0.5500E-02  0.1000E-06 NODE        7406   0.0%   6.845       22.10       0.000       28.95      0.1356      0.2446E-03  0.2920E-04
   60000  0.6000E-02  0.1000E-06 NODE        7406   0.0%   6.794       20.79       0.000       27.58      0.1356      0.2446E-03  0.2920E-04
   65000  0.6500E-02  0.1000E-06 NODE       68566  -0.1%   6.916       17.74      0.8650E-03   24.67      0.1357      0.2446E-03  0.2922E-04
   70000  0.7000E-02  0.1000E-06 NODE      162229  -0.1%   6.793       21.13      0.1916E-01   27.97      0.1357      0.2446E-03  0.2922E-04
   75000  0.7500E-02  0.1000E-06 NODE     1823082  -0.2%   7.149       20.41      0.2436E-01   27.63      0.1357      0.2446E-03  0.2922E-04

MESSAGE ID :        206

 ** RUN STOPPED: ENERGY ERROR LIMIT REACHED

   75380  0.7538E-02  0.1000E-06 NODE     1823387  99.9%   52.73       26.08      0.2593E-01   27.26      0.1357      0.2446E-03  0.2923E-04

      
                          ** STATISTICS   **
      
***  PARTS WITH HIGHEST KINETIC ENERGY
      
   PART_ID   K-ENERGY   PROP_ID PROP_TYPE    MAT_ID  MAT_TYPE
      
***  PARTS WITH HIGHEST INTERNAL ENERGY
      
   PART_ID   I-ENERGY   PROP_ID PROP_TYPE    MAT_ID  MAT_TYPE
      
***  NODES WITH HIGHEST VELOCITY
      
      NODE   VELOCITY     K-ENER       MASS      MASS0   DM/MASS0
   1823387 0.4792E+06  11.97     0.1043E-09 0.3325E-10  2.137    
   1821583 0.2802E+06  7.505     0.1911E-09 0.1651E-09 0.1574    
   1821994 0.5885E+05 0.1469     0.8485E-10 0.7481E-10 0.1342    
   1821993 0.4011E+05 0.8088E-01 0.1006E-09 0.7463E-10 0.3477    
   1821996 0.3860E+05 0.1034     0.1389E-09 0.9037E-10 0.5366    
   1821992 0.3830E+05 0.6138E-01 0.8370E-10 0.7291E-10 0.1479    
   1821991 0.3769E+05 0.5671E-01 0.7986E-10 0.7050E-10 0.1327    
   1821995 0.3760E+05 0.8832E-01 0.1249E-09 0.3220E-10  2.880    
   1821997 0.3336E+05 0.4984E-01 0.8958E-10 0.6883E-10 0.3015    
   1821998 0.2938E+05 0.4585E-01 0.1062E-09 0.7559E-10 0.4055    
      
***  NODES WITH HIGHEST MASS CHANGE
      
      NODE       MASS      MASS0   DM/MASS0
     15847 0.5024E-08 0.7095E-11  707.2    
     15732 0.2361E-08 0.3988E-11  591.1    
     15759 0.3837E-08 0.8633E-11  443.5    
     13761 0.2814E-08 0.6377E-11  440.3    
     14146 0.4888E-08 0.1187E-10  410.8    
     15807 0.4471E-08 0.1112E-10  401.2    
   1855609 0.2423E-08 0.6431E-11  375.8    
   1855694 0.3794E-08 0.1024E-10  369.5    
     15380 0.5124E-08 0.1601E-10  319.0    
   1855695 0.3040E-08 0.9859E-11  307.4

**Comment by PaulAltair** [comment]
I would look at node 1823387 here, it is what is reported as controlling the timestep at the termination: and has the highest velocity:

** RUN STOPPED: ENERGY ERROR LIMIT REACHED

75380 0.7538E-02 0.1000E-06 **NODE 1823387** 99.9% 52.73 26.08 0.2593E-01 27.26 0.1357 0.2446E-03 0.2923E-04

if you can plot velocity on your results, then maybe you can see where the issue is arising?

Pauly
